ARACHNE-2723 IR fails on BigQuery

YaroslavTir opened the pull request:
Fixes https://odysseusdataservices.atlassian.net/browse/ARACHNE-2723
- pass result schema as temp schema to the SqlRender
- replace deprecated translateSql method with translate

Commit:
ARACHNE-2723 IR fails on BigQuery
- pass result schema as temp schema to the SqlRender
- replace deprecated translateSql method with translate

wivern commented on R/run_ir_analysis.r:
```diff
@@ -18,7 +18,7 @@ run_ir_analysis <- function(basicDir, analysisId, cohortDefinitions, dbms, conne
 
   query <- SqlRender::readSql(system.file("sql/sql_server", "delete_strata.sql", package = "IncidenceRateSkeleton"))
   query <- SqlRender::renderSql(query, tableQualifier = resultsDatabaseSchema, analysis_id = analysisId)$sql
```
replace deprecated `renderSql` with newer `render`

wivern commented on R/run_ir_analysis.r:
```diff
@@ -18,7 +18,7 @@ run_ir_analysis <- function(basicDir, analysisId, cohortDefinitions, dbms, conne
   query <- SqlRender::readSql(system.file("sql/sql_server", "delete_strata.sql", package = "IncidenceRateSkeleton"))
   query <- SqlRender::renderSql(query, tableQualifier = resultsDatabaseSchema, analysis_id = analysisId)$sql
-  query <- SqlRender::translateSql(query, targetDialect = connectionDetails$dbms)$sql
+  query <- SqlRender::translate(query, targetDialect = connectionDetails$dbms, oracleTempSchema=resultsDatabaseSchema)
```
I'd like to suggest to add `tempDatabaseSchema` as an optional parameter with `resultsDatabaseSchema` as default

Commit:
ARACHNE-2723 IR fails on BigQuery (fix PR remarks)
- use render instead of deprecated renderSql
- add tempDatabaseSchema as default argument in method
